src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10508,6 +10508,7 @@ public sealed partial class SecurityElement
         public string? Attribute(string name) { throw null; }
         public System.Security.SecurityElement Copy() { throw null; }
         public bool Equal([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] System.Security.SecurityElement? other) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNull("str")]
```

Comment:
```suggestion
        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("str")]
```

(see example on line 10510 above or line 10514 below: we tend to use the full type name, including namespace _and Attribute suffix_)

Reply:
Oops, third time's the charm